bin/run-nextclade-full

```diff
@@ -223,36 +233,88 @@ main() {
+    csvstack --tabs ${TSV_OUTPUT_WILDCARD} | csvformat --out-tabs >"${OUTPUT_TSV}"
   else
-    echo "[ERROR] ${0}:${LINENO}: Output batches are not found: '${OUTPUT_WILDCARD}'. Nextclade jobs mush have failed to write their results."
+    echo "[ERROR] ${0}:${LINENO}: Output TSV batches are not found: '${TSV_OUTPUT_WILDCARD}'. Nextclade jobs mush have failed to write their results."
     exit 1
   fi
 
-  # Check if output batches exist and report
+  # Check if output TSV result exist
   # shellcheck disable=SC2086 # We want globbing here
   if ! ls ${OUTPUT_TSV} 1>/dev/null 2>&1; then
-    echo "[ERROR] ${0}:${LINENO}: Concatenated result not found: '${OUTPUT_TSV}'. Concatenation of output batches must have failed."
+    echo "[ERROR] ${0}:${LINENO}: Concatenated TSV result not found: '${OUTPUT_TSV}'. Concatenation of output TSV batches must have failed."
+    exit 1
+  fi
+
+  # Check if output FASTA batches exist and report
+  # shellcheck disable=SC2086 # We want globbing here
+  if ls ${FASTA_OUTPUT_WILDCARD} 1>/dev/null 2>&1; then
+    # shellcheck disable=SC2012 # We want globbing here
+    NUM_OUTPUT_FASTA_BATCHES="$(ls -Ubad1 -- ${FASTA_OUTPUT_WILDCARD} 2>/dev/null | wc -l)"
+    echo "[ INFO] ${0}:${LINENO}: There are now ${NUM_OUTPUT_FASTA_BATCHES} output FASTA batches to concatenate (batch size is ${BATCH_SIZE})"
+    echo "[ INFO] ${0}:${LINENO}: Concatenating Nextclade output FASTA batches from '${FASTA_OUTPUT_WILDCARD}' into '${OUTPUT_FASTA}'"
+    cat ${FASTA_OUTPUT_WILDCARD} >"${OUTPUT_FASTA}"
+  else
+    echo "[ERROR] ${0}:${LINENO}: Output FASTA batches are not found: '${FASTA_OUTPUT_WILDCARD}'. Nextclade jobs mush have failed to write their results."
+    exit 1
+  fi
+
+  echo "[ INFO] ${0}:${LINENO}: Calculate mutation_summary.tsv for every batch"
+  for input in ${INPUT_WILDCARD}; do
+    if [ ! -e "${input}" ]; then
+      # If ${input} does not exist, this means that INPUT_WILDCARD was not expanded,
+      # there are no batch files and we should not attempt to run the processing.
+      # If we would, then it will crash.
+      break
+    fi
+
+    input_basename="$(basename "${input}")"
+    output_fasta_basename="${input_basename%.fasta}.aligned.fasta"
+    output_fasta_filename="${TMP_OUTPUT_FASTA_DIR}/${output_fasta_basename}"
+    output_dir="${TMP_NEXTCLADE_OUTPUT_DIR}/${input_basename%.fasta}"
+    output_mutation_summary="${TMP_OUTPUT_MUTATION_SUMMARY_DIR}/${input_basename%.fasta}.mutation_summary.tsv"
+
+    mkdir -p "${TMP_OUTPUT_MUTATION_SUMMARY_DIR}"
+
+    # Note we need genes variable to be unquoted
+    # shellcheck disable=SC2086
+    ./bin/mutation-summary \
+      --directory="${output_dir}" \
+      --alignment="${output_fasta_filename}" \
+      --basename="nextclade" \
+      --genes ${GENES_SPACE_DELIMITED} \
+      --reference="${TMP_DIR_NEXTCLADE_DATASET}/reference.fasta" \
+      --genemap="${TMP_DIR_NEXTCLADE_DATASET}/genemap.gff" \
+      --output="${output_mutation_summary}"
```

Comment:
I also silenced the [SC2086](https://github-wiki-see.page/m/koalaman/shellcheck/wiki/Sc2086) warning for `GENES_SPACE_DELIMITED`. We need  it to be
```
--genes A B C
```
without quotations marks.